pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -21,50 +36,491 @@ func init() {
 	defaultEnforcer = regoEnforcer
 }
 
+//go:embed framework.rego
+var FrameworkCode string
+
+//go:embed policy.rego
+var PolicyCode string
```

Comment:
Do `FrameworkCode` and `PolicyCode` need to be exported?

Reply:
Same with `Indent` below.

Reply:
done